packages/oui-select/README.md

```diff
@@ -68,6 +68,22 @@
 </oui-select>
 ```
 
+### Disabled Items
+
+```html:preview
+<oui-select name="country"
+    model="$ctrl.country"
+    data-title="Select a country"
+    placeholder="Select a country..."
+    items="$ctrl.countries"
+    disable-items="$ctrl.disableItems($item)"
```

Comment:
What's behind `$ctrl.disableItems($item)` (without looking on specs, just by reading the docs) ? 
I think we should add a note or an example that explain how this function work (since it's a filter).

Also, don't you have a PR to link with `ovh-ui-kit-documentation` ?